src/job/stagingJobHandler.ts

```diff
@@ -55,7 +55,9 @@ export class StagingJobHandler extends JobHandler {
   prepStageSpecificNextGenCommands(): void {
     if (this.currJob.buildCommands) {
       this.currJob.buildCommands[this.currJob.buildCommands.length - 1] = 'make next-gen-html';
-      this.currJob.buildCommands.push(`make oas-page-build MUT_PREFIX=${this.currJob.payload.mutPrefix}`);
+      this.currJob.buildCommands.push(
```

Comment:
these give me the hives on grounds of potential command line injection concerns, but that's how the code is structured, so *chews fingernails nervously* it's okay for now

Reply:
This is such a good point. I can create a check on PROJECT because it should only be cloud-docs that's ever needed... for now. Branch name is harder. I can add a regex check to make sure it can only have the characters allowed in a github branch name, a-zA-Z0-9_/-.

Reply:
@i80and - the cloud-docs check might be a little silly, but these additions make my mind more at ease. Lmk what you think!

Reply:
I'm good with this! The real solution is to not invoke a shell, but this validation is much better than nothing!